system_tests/common_test.go

```diff
@@ -1424,7 +1406,7 @@ func Create2ndNodeWithConfig(
 	Require(t, err)
 	wasmData, err := chainStack.OpenDatabaseWithExtraOptions("wasm", 0, 0, "wasm/", false, conf.PersistentConfigDefault.Pebble.ExtraOptions("wasm"))
 	Require(t, err)
-	chainDb := rawdb.WrapDatabaseWithWasm(chainData, wasmData, wasmCacheTag, execConfig.StylusTarget.WasmTargets())
+	chainDb := rawdb.WrapDatabaseWithWasm(chainData, wasmData)
```

Comment:
as now the wasm tag is coupled with `MessageRunContext`: are we ok with the all tests running with wasmCacheTag with wasm long term cache enabled?

Reply:
The problem with tests using long-term cache is that multiple tests run in parallel, and in prallel with other tests initializing static vars etc. It's.. probably o.k. as long as the tests that do target long-term cache are not run in parallel with anything else.